ad-remover.css

```diff
@@ -7,9 +7,17 @@
 }
 
 .main {
-	padding-top: 9em !important;
+	padding-top: 8em !important;
 }
 
 .page-results {
-	padding-bottom: 1em !important
+	padding: 0.5em 0 !important
+}
+
+.page-results .item {
+	margin-bottom: 1.6em !important;
```

Comment:
Currently it's about 2.1em

Reply:
Funny how I was thinking about the same!